docs/api/classifier/class_prediction_error.rst

```diff
@@ -19,13 +23,39 @@ The class prediction error chart provides a way to quickly understand how good y
     # Perform 80/20 training/test split
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20,
                                                         random_state=42)
+    # Instantiate the classification model and visualizer
+    visualizer = ClassPredictionError(
+        RandomForestClassifier(), classes=classes
+    )
 
-.. code:: python
+    # Fit the training data to the visualizer
+    visualizer.fit(X_train, y_train)
 
-    from sklearn.ensemble import RandomForestClassifier
+    # Evaluate the model on the test data
+    visualizer.score(X_test, y_test)
 
+    # Draw visualization
+    visualizer.poof()
+
+
+.. plot::
```

Comment:
Should this 2nd plot-directive be included?

Reply:
@lwgray I like the idea of including (and showing) both plots! One thought; would you be open to adding a few lines of interpretation to each?